bika/lims/browser/reports/administration_usershistory.py

```diff
@@ -159,7 +160,7 @@ def __call__(self):
                                     'Workflow': '',
                                     'Action': metatitle,
                                     'ActionDate': meta['timestamp'],
-                                    'ActionDateStr': meta['timestamp'],
+                                    'ActionDateStr': self.ulocalized_time(action_date, 1),
```

Comment:
I think the `1` is for `long_format`? Please add the keyword instead of a positional argument